src/main/scala/com/codacy/analysis/cli/command/analyse/AnalyseExecutor.scala

```diff
@@ -138,6 +145,13 @@ class AnalyseExecutor(toolInput: Option[String],
     }
   }
 
+  private def overrideFilePermissions(sourceDirectory: File) = {
+    Process(Seq("find", sourceDirectory.pathAsString, "-type", "d", "-exec", "chmod", "u+rwx", "{}", ";")).!
+    Process(Seq("find", sourceDirectory.pathAsString, "-type", "d", "-exec", "chmod", "ugo+rx", "{}", ";")).!
+    Process(Seq("find", sourceDirectory.pathAsString, "-type", "f", "-exec", "chmod", "u+rw", "{}", ";")).!
+    Process(Seq("find", sourceDirectory.pathAsString, "-type", "f", "-exec", "chmod", "ugo+r", "{}", ";")).!
+  }
```

Comment:
Maybe move this to codacy-analysis-cli/src/main/scala/com/codacy/analysis/cli/utils/FileHelper.scala's FileHelper object

Reply:
I do not want this to be used by anyone else neither in other places.